content/docs/04-clusters/06-cluster-management/05-backup-restore/01-add-backup-location.md

```diff
@@ -0,0 +1,357 @@
+
+  If you choose the Credentials method, provide an IAM user's access key. The IAM user must have the above permissions to allow Palette create a backup in the S3 bucket. You can use the following steps to create a new IAM user and generate the access key. <br /> <br />
+	- Switch to the AWS console. Use the AWS account that hosts the S3 bucket.
+	- Navigate to the IAM service, and select **Users** from the left **Main Menu**. 
+	- Click on the **Create user** button to create a new user. 
+	- Provide a name for the user. Ensure that the user has programmatic access.
+	- Attach the IAM policy you defined above to the IAM user. Alternatively, you can add it as an inline policy. 
+	- Review the user permissions and finish creating the user. 
+	- AWS will generate and display the access key for the newly created user. Copy the newly generated access key id and secret access key, and paste them into Palette.
+
+  </Tabs.TabPane>
+
+  <Tabs.TabPane tab="STS" key="sts">
+
+  Palette supports the STS credentials method for adding only the AWS cloud as the backup location. Palette does not support the STS credentials method for adding other backup location providers, such as Azure, GCP, or MinIO. 
+	
+	If you use the STS method, you must create a new IAM role and provide its ARN. The IAM role must have the necessary IAM permissions defined above and a trust relationship. You can use the following steps to create the IAM role. <br /> <br />
+	- Switch to the AWS console. Use the AWS account that hosts the S3 bucket.
+	- Navigate to the IAM service, and select **Roles** from the left **Main Menu**. 
+	- Click on the **Create role** button to create a new role. 
+	- Use the following configuration while creating the IAM role in the AWS console. 
+
+    |**AWS Console Field**|**Value**|
+    |---|---|
+    |Trusted entity type| AWS account.|
+    |An AWS account|Select the Another AWS account radio button.|
+    |AWS Account ID|Use the one displayed in Palette. It is Palette SaaS account ID.|
+    |Options|Select the Require external ID checkbox.|
+    |External ID|Use the one displayed in Palette. Palette generates the external ID.|
+    |Permissions policies| Attach the IAM policy defined above.|
+    |Role name|Your choice.|
+    |Role description|Your choice.|
+    
+	- Review the trusted entities and the attached permissions. 
+	- Finish creating the IAM role.
+	- Copy the IAM role ARN and paste it into Palette.
+
+  </Tabs.TabPane>
+
+	</Tabs>
+
+
+
+6. Click on the **Validate** button after providing the IAM user's access key or the IAM role ARN. 
+
+
+7. If the backup cloud account differs from the one where Palette deploys the clusters. In such a scenario, you must also authorize the latter cloud account. You must update the existing trust policy by appending the following permission. The permission below will authorize the AWS account where Palette deploys the clusters to assume the current IAM role. <br /> <br />	
```

Comment:
Combining a simple scenario with a more complicated one is a bad idea. I would break this out into another how-to below this one that focuses on multiple-accounts. Otherwise, you increase the reading difficulty for the reader.